### PR TITLE
Add twilio call and sendSMS method

### PR DESCRIPTION
**Pull request by icltc**

### CODE REVIEW

File: back.py [hunk 1]
```diff
@@ -38,6 +44,24 @@ def play(file):
     soundPlayStop = True
     soundPlaying = False
 
+def call(number):
+    client = Client(account_sid, auth_token)
```
Comment (HelloDaF) [inline back.py]: Could you please rename it to TwilioClient to specify?

File: back.py [hunk 2]
```diff
@@ -38,6 +44,24 @@ def play(file):
...
+    call = client.calls.create(
+                        url='http://demo.twilio.com/docs/voice.xml',
+                        to=number,
+                        from_=source_phone_number
+                    )
+
+def sendSMS(number, message):
+    client = Client(account_sid, auth_token)
```
Comment (HelloDaF) [inline back.py]: is the client can be initialised in the beginning?

File: back.py [hunk 3]
```diff
@@ -38,6 +44,24 @@ def play(file):
     soundPlayStop = True
     soundPlaying = False
 
+def call(number):
+    client = Client(account_sid, auth_token)
+    call = client.calls.create(
+                        url='http://demo.twilio.com/docs/voice.xml',
+                        to=number,
+                        from_=source_phone_number
```
Comment (HelloDaF) [inline back.py:65]: We need more logs here to monitor the attempt calls

File: back.py (hunk 2 above)
Comment (icltc) [reply back.py]: yes, but it is not very expensive to create this client, will leave it here.

File: back.py (hunk 3 above)
Comment (icltc) [reply back.py:65]: sure!

**Review by HelloDaF — APPROVED** [review]

**Pull request merged**